OrbitClientModel/CaptureDeserializer.cpp

```diff
@@ -148,6 +148,28 @@ void LoadCaptureInfo(const CaptureInfo& capture_info, CaptureListener* capture_l
     capture_listener->OnCallstackEvent(std::move(callstack_event));
   }
 
+  for (const orbit_client_protos::TracepointInfo& tracepoint_info :
+       capture_info.tracepoint_infos()) {
+    if (*cancellation_requested) {
+      capture_listener->OnCaptureCancelled();
+      return;
+    }
+    orbit_grpc_protos::TracepointInfo tracepoint_info_translated;
```

Comment:
It feels a bit strange that you are using a grpc proto for `TracepointInfo` on the Listener, and a client proto for `TracepointEventInfo`, but I see that it is the same for callstacks. So I guess you can leave it as is for now.

Reply:
I kind of agree that we want to reuse grpc protos where possible. But this is a separate task. Currently client protos are completely independent from grpc ones.

Reply:
This is more related to the mechanism used to send data from the service to the client. If you want we can discuss this further in more detail.